aurora/main.tf

```diff
@@ -1,38 +1,42 @@
+terraform {
```

Comment:
Can you move this to a `versions.tf`  file? It's how the migration tool sets it and we're keeping it that way in other modules we've migrated